registry/registry-options.adoc

```diff
@@ -14,6 +14,8 @@ include::modules/registry-integrated-openshift-registry.adoc[leveloffset=+1]
 
 include::modules/registry-third-party-registries.adoc[leveloffset=+1]
 
+include::modules/registry-authentication.adoc[leveloffset=+1]
```

Comment:
To match the original TOC, I would recommend changing the `leveloffset` value to `leveloffset=+2`